.circleci/config.yml

```diff
@@ -40,7 +41,9 @@ test_job: &test_job
         at: .
     - macos/preboot-simulator:
         # Keep simulator in sync with `Library/TestHelpers/TestCase.swift` and `Makefile`
-        version: "17.2"
+        # iOS 17.5 is minimum supported runtime by circle-ci -> 16.0 (16A242d)
+        # more info: https://circle-macos-docs.s3.amazonaws.com/image-manifest/v15048/manifest.txt
+        version: "17.5"
```

Comment:
I'm glad this didn't require re-recording snapshots - I was worried about that. Did you check if updating to 18.0 would also pass tests without any further work needed? If they're equally convenient, I think we may want to update to iOS 18 instead. (Optional; we could also update to iOS 18 later!)

Reply:
Yup, I also tried with iOS 18 so definitely we need to re-recording the snapshots :/, do we need a migration plan?

Reply:
Thanks for clarification.